site/components/src/main/java/uk/nhs/hee/web/constants/HEETaxonomy.java

```diff
@@ -6,7 +6,7 @@ public enum HEETaxonomy {
 
     HEE_GLOBAL_HEALTHCARE_TOPICS ("GlobalHealthcareTopics"),
 
-    HEE_GLOBAL_PUBLICATION_TYPE ("GlobalPublicationType");
+    HEE_GLOBAL_PUBLICATION_TYPE ("GlobalPublicationTypes");
```

Comment:
Should now be HEE_GLOBAL_PUBLICATION_TYPES

Reply:
Sure, thanks. This has now been resolved 👍